src/components/links/__tests__/Link-test.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { screen } from '@testing-library/react';
+import { MemoryRouter, Route, Routes, useLocation } from 'react-router-dom';
+import { LinkSubdued } from '..';
+import { render } from '../../../common/helpers/testUtils';
+import { Link } from '../Link';
+
+beforeAll(() => {
+  const { location } = window;
+  delete (window as { location?: Location }).location;
+  window.location = { ...location, href: '' };
+});
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+// This functionality won't be needed once we update the breadcrumbs
```

Comment:
I wonder if we should keep this comment since we don't have a breadcrumb component